source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -20,7 +20,7 @@ async function loadDeferred(jumpList: Element): Promise<void> {
 
 async function init(): Promise<void | false> {
 	const fileList = await elementReady([
-		'.toc-select details-menu', // `isPR`
+		'.toc-select details-menu:not([src$="conversations_menu"])', // `isPR`
```

Comment:
This seems to work on this PR:


```suggestion
		'.toc-select details-menu[src*="/show_toc?"]', // `isPR`
```

The attribute is:

```
src="/sindresorhus/refined-github/pull/4401/show_toc?base_sha=a3201546e68e7da87fa9f3ad7dc28354c80c058f&sha1=a3201546e68e7da87fa9f3ad7dc28354c80c058f&sha2=72ee464b18951ebc582fe0224daf8920a5e1fd24"
```